lib/services/sf.js

```diff
@@ -1378,11 +1378,14 @@ StateConnected.prototype.request = function (options)
  */
 StateConnected.prototype.destroy = function (options)
 {
-  // send out a master token request to terminate the current connection
-  this.createMasterTokenRequest(
+  var requestID = uuidv4();
+  var requestGUID = uuidv4();
+
+  // send out a session token request to terminate the current connection
+  this.createSessionTokenRequest(
     {
       method: 'POST',
-      url: '/session/logout-request',
+      url: `/session?delete=true&requestId=${requestID}&request_guid=${requestGUID}`,
```

Comment:
Where do we replace the master token with session token? the `createSessionTokenRequest` function would do that for us?

Reply:
Yes, the createSessionTokenRequest() function will use the session token instead of the master token for the Snowflake Token header value